src/types.ts

```diff
@@ -2063,3 +2063,62 @@ export type User<UserType = UnknownType> = UserType & {
   teams?: string[];
   username?: string;
 };
+
+export type SegmentData = {
+  description: string;
+  filter: {
+    channel?: object;
+    user?: object;
+  };
+  name: string;
+};
+
+export type Segment = {
+  app_pk: number;
+  created_at: string;
+  id: string;
+  updated_at: string;
+  recipients?: number;
+} & SegmentData;
+
+export type CampaignData = {
+  attachments: Attachment[];
+  defaults: { [key: string]: string };
+  segment_id: string;
+  sender_id: string | null;
```

Comment:
should this be optional to be omitted from the request?

Reply:
Yeah, so I guess it should be `string?` instead. I sthat what you're suggesting?

Reply:
exactly